packages/backend/src/helpers/generate-step-error.ts

```diff
@@ -0,0 +1,40 @@
+import HttpError from '@/errors/http'
+import StepError from '@/errors/step'
+
+const appKeyToActionMapping: Record<string, string> = {
```

Comment:
can we pull this from the app definition instead of having to maintain this separately?

Reply:
will do so after the pr for typing gets merged!

Reply:
eh i realise dont need the step app key anymore, can use app name